hips/draw/simple.py

```diff
@@ -1,5 +1,152 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
+import os
+from pathlib import Path
+
+import numpy as np
+from astropy.coordinates import SkyCoord
+from skimage import transform as tf
+from typing import List
+
+from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
+from ..utils import WCSGeometry, compute_healpix_pixel_indices, boundaries
 
 __all__ = [
+    'draw_sky_image',
+    'make_sky_image',
+    'SimpleTilePainter'
 ]
+
+
+def draw_sky_image(geometry: WCSGeometry, tiles: List[HipsTile]) -> np.ndarray:
+    """Draw sky image using the simple and quick method.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tiles : List[HipsTile]
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    all_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        draw_tile = SimpleTilePainter(geometry, tile, all_sky.shape)
+        draw_tile.compute_corners()
+        draw_tile.compute_projection()
+        all_sky += draw_tile.warp_image()
+    return all_sky
+
+
+class SimpleTilePainter:
+    """A class which iteratively draws a tile following the naive algorithm steps mentioned `here <https://hips.readthedocs.io/en/latest/drawing_algo.html#naive-algorithm>`_.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tile : `HipsTile`
+       An object of HipsTile
+    shape : tuple
+        Shape of the all-sky image
+    """
+
+    def __init__(self, geometry: WCSGeometry, tile: HipsTile, shape: tuple) -> None:
+        self.geometry = geometry
+        self.tile = tile
+        self.shape = shape
+        self.corners = None
+        self.pt = None
+
+    def compute_corners(self) -> None:
+        theta, phi = boundaries(self.tile.meta.nside, self.tile.meta.ipix)
+        radec = SkyCoord(ra=phi, dec=np.pi / 2 - theta, unit='radian', frame='icrs')
```

Comment:
Here you hardcode `frame='icrs'`. But tiles can be in Galactic frame.
So here you should instead add the frame as an attribute to `HipsTileMeta` and then access it from here.